src/app/project/beta/ontology-classes/ontology-class-item/ontology-class-item.component.ts

```diff
@@ -58,12 +60,25 @@ export class OntologyClassItemComponent implements OnInit {
                 this._errorHandler.showMessage(error);
             }
         );
+
+        this.icon = this._getIcon();
     }
 
     open(route: string) {
         this._router.navigateByUrl(route);
     }
 
+    /**
+     * given an Html element, compare the scrollHeight and the clientHeight
+     * determines whether or not the tooltip should be displayed
+     *
+     * @param elem the element which has the line-clamp css
+     * @returns inverse of comparison between the scrollHeight and the clientHeight of elem
+     */
+    hideTooltip(elem: HTMLElement): boolean {
+        return !(elem.scrollHeight > elem.clientHeight);
+    }
```

Comment:
I would rename the function because it does not directly do something about tool tip. It compares the height of the elements. So I would use something like "compareHeights" or "checkHeights" or "checkElementHeight" etc.

Reply:
good point! done in [1b17e0b](https://github.com/dasch-swiss/dsp-app/pull/790/commits/1b17e0b73acd424ad941d9f83fa89f51660dcd5e)